lib/takeaway.rb

```diff
@@ -1,8 +1,7 @@
 require 'twilio-ruby'
 
 # put your own credentials here
-account_sid = 'AC71ede767f3f69bf43718d8094b244800'
-auth_token = '88ac7f43909c00b5b8e9a32d5b554c35'
+#taken them out for pull request
```

Comment:
Deleting them doesn't quite work for security - I can still find them here! Once you have got feedback from a coach on this repo, I'd consider making it a private repo.